src/terragraph-e2e/e2e/config/config_metadata.json

```diff
@@ -533,6 +533,14 @@
         "regexMatches": "|(([A-Za-z0-9_]+=-?[0-9]+)(,[A-Za-z0-9_]+=-?[0-9]+)*)"
       }
     },
+    "SERIAL_CONSOLE_DISABLE": {
+      "desc": "(DANGER!) Disable the serial console on the node",
```

Comment:
nit. Suggest more explicit description.

`"desc": "(DANGER!) Disable the linux serial console (/dev/ttyS0) on the node",`

Reply:
Will add "Linux" but would rather omit the device name here (might not be ttyS0 on all hardware types, and implementation does not care which tty).